### ISSUE TITLE
SublimeLinter-flow doesn't respect a @FlowFixMe comment

### ISSUE THREAD

**Issue by jdalegonzalez** [SublimeLinter/SublimeLinter-flow#50]
If the following line is in a file, you'll see a flow error warning about userSelect not existing on the style object

```var es = (document.documentElement) ? document.documentElement.style.userSelect : null;```

The following change should cause the error to go away...

```
// $FlowFixMe: Flow is wrong.  userSelect exists.  At least in Chrome world.
var es = (document.documentElement) ? document.documentElement.style.userSelect : null;
```

**Comment by davidaurelio** [comment]
I believe the issue is that flow does not support error suppressing comments with the `check-contents` command.

**Comment by braver** [comment]
Well, that makes it a flow bug, not something particular to this repo right? We could change the behaviour to not use stdin, but what use is supporting stdin in your linter if it's going to ignore config like this?
